solr-template/build.gradle

```diff
@@ -0,0 +1,77 @@
+plugins {
+    id "java"
+    id "com.bmuschko.docker-remote-api" version "6.4.0"
```

Comment:
Apply `eu.xenit.docker-config` so the `docker-remote-api` plugin is configured correctly